src/components/interaction-designer/toolbar/TreeBuilderToolbar.vue

```diff
@@ -368,6 +370,17 @@ export class TreeBuilderToolbar extends mixins(Routes, Permissions, Lang) {
   isExportVisible = false
   height = 102
 
+  created(): void {
+    const $store = this.$store
+    const moduleName = 'undoRedo'
+
+    if (!$store.hasModule(moduleName)) {
+      $store.registerModule(moduleName, undoRedoModule)
+    }
+
+    VuexUndoRedoPlugin(this.$store)
+  }
+
```

Comment:
Would it be a good idea to extract this logic into a separate function inside `vuex-undo-redo-plugin.ts` file, because `TreeBuilderToolbar.vue` is pretty big?

Reply:
Wait, this is also registering the module that is in a separate file. Never mind